src/support.hbs

```diff
@@ -0,0 +1,52 @@
+---
+title: Get in Touch!
+---
+<section class="general section is-revealing">
+	<div class="general-inner section-inner">
+		<div class="container-sm">
+			<h1>{{title}}</h1>
+			{{!-- <p>Explore our guides to make the most of Gradebook</p> --}}
+			<p>Questions? Comments? We'd love to talk!</p>
+		</div>
+		<div class="container-sm">
+			<form class="support-form container-xs section-inner" netlify>
+				<input type="hidden" name="school" value="none" />
+				<p>
+					<label for="name">Name</label>
+					<input autofocus type="text" name="name" required />
+				</p>
+				<p>
+					<label for="email">Email Address</label>
+					<input type="email" name="email" required />
+				</p>
+				<p>
+					<label for="subject" class="inline">What are you looking for?</label>
+					<select name="subject">
+						<option value="support">Get help using Gradebook</option>
+						<option value="bug_report">Report an issue</option>
+						<option value="feature_request">Suggest a feature</option>
+						<option value="partnership">Discuss a partnership</option>
+						<option value="other">Something else</option>
+					</select>
```

Comment:
I think the select dropdown might look better on a new line

Reply:
Main reason for keeping it on the same line is for the `Something else` - keeping the select on a separate line causes some grouping issues when it's visible